native/app/Source/Application.swift

```diff
@@ -275,7 +275,7 @@ class Application {
     Driver.device!.setVirtualMasterVolume(volume > 1 ? 1 : Float32(volume), direction: .playback)
     Driver.latency = selectedDevice.latency(direction: .playback) ?? 0 // Set driver latency to mimic device
     Driver.safetyOffset = selectedDevice.safetyOffset(direction: .playback) ?? 0 // Set driver latency to mimic device
-    self.matchDriverSampleRateToOutput()
+    self.matchDriverSampleRateTo48000()
```

Comment:
Why are you pinning the driver sample rate to 48kHz instead of allowing the driver to run at closest sampling rate to the output device? In theory the closer the sampling rate is the less audio quality will be lost in the pipeline?

Reply:
Because then EQ works correctly on different sampling rates. Without it on 16 Khz EQ also moves to lower frequencies and settings become incorrect (treble becomes bass).

Reply:
Of course it's better to change EQ processing logic to considers sample rate changes and shift frequencies as well, but I'm new to this and don't fully know how it works yet. I think you can plan these changes and we will do it later.